File: app/src/main/java/com/codepath/bestsellerlistapp/models/BestSellerResults.java
package com.codepath.bestsellerlistapp.models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class BestSellerResults {

    @SerializedName("list_name")
    public String listName;

    @SerializedName("books")
    public List<BestSellerBook> books;
}

File: app/src/main/java/com/codepath/bestsellerlistapp/models/BestSellerResults.kt
package com.codepath.bestsellerlistapp.models

import com.google.gson.annotations.SerializedName

class BestSellerResults {
    @SerializedName("list_name")
    var listName: String? = null

    @JvmField
    @SerializedName("books")
    var books: List<BestSellerBook>? = null
}